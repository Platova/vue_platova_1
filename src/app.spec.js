import {describe, it, expect,vi} from "vitest";
import {shallowMount} from "@vue/test-utils"
import component from "./App.vue"


describe('App component', () => {


    it ('Mount without error', () => {
        const wrapper = shallowMount(component, {
            global: {
                stubs:  {
                    'RouterView': { template: '<div/>' },
                }
            }
        })
        expect(wrapper.exists()).toBeTruthy();
        expect(wrapper.html()).toMatchSnapshot();
    })
})