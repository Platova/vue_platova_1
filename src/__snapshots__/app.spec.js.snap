// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > Mount without error 1`] = `
"<div id=\\"app\\">
  <app-header-stub></app-header-stub>
  <div style=\\"padding-top: 5rem; text-align: center;\\">
    <div></div>
  </div>
</div>"
`;
